packages/ui/src/composites/wui-transaction-visual/index.ts

```diff
@@ -0,0 +1,54 @@
+import { html, LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import { elementStyles, resetStyles } from '../../utils/ThemeUtil'
+import '../wui-icon-box'
+import '../../components/wui-image'
+import styles from './styles'
+import type { TransactionIconType, TransactionType } from '../../utils/TypesUtil'
+
+@customElement('wui-transaction-visual')
+export class WuiTransactionVisual extends LitElement {
+  public static styles = [resetStyles, elementStyles, styles]
+
+  // -- State & Properties -------------------------------- //
+  @property() public type: TransactionType = 'buy'
+
+  @property() public imageSrc = ''
+
+  // -- Render -------------------------------------------- //
+  public render() {
+    let color: 'blue-100' | 'error-100' | 'success-100' = 'blue-100'
+    let icon: TransactionIconType = 'arrowTop'
+    if (
```

Comment:
You can make small arrays here i.e `const outgoing = ['bought', 'buy', 'deposited']` etc. and then use them in if statements like `if (outgoing.includes(this.type))`, should help simplify thinsg